Dockerfile

```diff
@@ -3,7 +3,7 @@
 ##
 ## Build
 ##
-FROM golang:1.17.7-alpine AS build
+FROM golang:1.18.7-alpine AS build
```

Comment:
I believe there's a `1.18.8` version now. You might want to use that one.

Reply:
I think is because the other version also has some vulnerabilities.
![image](https://user-images.githubusercontent.com/32104859/207628948-7da31504-2740-4daf-938f-2b5fbba73e32.png)